### PR TITLE
Use bundled tar kubectl for integration tests. Add log verbosity args.

### PR DESCRIPTION
**Pull request by davidz627**
/assign @msau42
/assign @krzyzacy

### CODE REVIEW

File: test/run-k8s-integration.sh [hunk 1]
```diff
@@ -13,6 +13,7 @@ readonly PKGDIR=${GOPATH}/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver
 readonly overlay_name="${GCE_PD_OVERLAY_NAME:-prow-head-template}"
 readonly boskos_resource_type="${GCE_PD_BOSKOS_RESOURCE_TYPE:-gce-project}"
 readonly do_driver_build="${GCE_PD_DO_DRIVER_BUILD:-true}"
+export GCE_PD_VERBOSITY=9
```
Comment (krzyzacy) [inline test/run-k8s-integration.sh]: should this also be configureable?

File: test/k8s-integration/main.go [hunk 2]
```diff
@@ -148,6 +148,12 @@ func handle() error {
 			return fmt.Errorf("failed to build Kubernetes: %v", err)
 		}
 
+		// Set kubectl to the one bundled in the k8s tar for versioning
+		err = os.Setenv("GCE_PD_KUBECTL", filepath.Join(k8sDir, "cluster", "kubectl.sh"))
```
Comment (krzyzacy) [inline test/k8s-integration/main.go]: I think you also want to check if `k/cluster/kubectl.sh` actually exists

File: test/run-k8s-integration.sh (hunk 1 above)
Comment (davidz627) [reply test/run-k8s-integration.sh]: for integration tests we always want to just set it to max. noisy logs for our tests is fine and helps with debugging

**Comment by davidz627** [comment]
thanks @krzyzacy resolved your comments

**Pull request merged**